src/components/augmentedReality/AugmentedReality.js

```diff
@@ -39,6 +39,10 @@ export const AugmentedReality = ({ navigation, onSettingsScreen }) => {
     setIsLoading(false);
   };
 
+  if (error) return null;
```

Comment:
what about using our `EmptyMessage` component here for users instead of a white screen?

Reply:
what about using our EmptyMessage component here for users instead of a white screen?

Reply:
a blank page is not displayed here. Only the Augmented section is not displayed.

Reply:
ah jo, good!